src/styles/less/JoinTheSquad.less

```diff
@@ -49,47 +49,178 @@
 .sharePoints {
   background: #eb7d5b;
   border-color: #eb7d5b;
-  margin-bottom: 3%;
-  margin-right: 3%;
+  // margin-bottom: 3%;
+  // margin-right: 3%;
+  position:fixed;
+  bottom: 30px;
+  right: 30px;
   &:hover {
     background-color: #eb7d5bbb;
     border-color: #eb7d5bbb;
   }
+  @media @mobile{
+    position: relative;
+    right:56px;
+    bottom: 57.5vh;
+    height: 5vh;
+    width: 218px;
+  }
 }
 img {
   max-width: 100%;
 }
 
 .star {
-  z-index: 1;
-  margin-top: 20%;
+  z-index: 2;
+  // margin-top: 20%;
+  position: inherit;
+  bottom:95px;
+  @media @mobile{
+    height: 32vh;
+  }
 }
 
 .wordBubble {
-  z-index: 2;
-  position: fixed;
+  z-index: 3;
+  // position: fixed;
+  margin-left: 12%;
+  margin-top: 0%;
+  @media @mobile{
+    margin-top: 33%;
```

Comment:
awesome implementation of media queries!

Reply:
That z-index positioning was a challenge. Good job figuring that out with @manzur1990